dumper.sh

```diff
@@ -554,6 +580,8 @@ done
 
 # Process All partitions From TMPDIR Now
 for partition in ${PARTITIONS}; do
+	foundpart=$(7z l -ba "${FILEPATH}" | gawk '{print $NF}' | grep "${partition}.img")
+	7z e -y -- "${FILEPATH}" "${foundpart}" dummypartition 2>/dev/null
```

Comment:
This part is missing in the `superimage_extract` function, will re-configure this part.

Reply:
Ya that's why added out of all conditions it will work in almost every case I think

Reply:
It was already a part of superimage extraction function, but now it can be used universally.